datafusion/physical-plan/src/aggregates/row_hash.rs

```diff
@@ -102,6 +102,19 @@ struct SpillState {
 
     /// true when streaming merge is in progress
     is_stream_merging: bool,
+
+    // ========================================================================
+    // METRICS:
+    // ========================================================================
+    /// Peak memory used for buffered data.
+    /// Calculated as sum of peak memory values across partitions
+    peak_mem_used: metrics::Gauge,
+    /// count of spill files during the execution of the operator
+    spill_count: metrics::Count,
```

Comment:
`spill_files` or ``spill_file_count` ?

Reply:
We should also keep the name consistent in other operators, I am thinking about a related refactor, so I can do this in another PR